docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -175,7 +159,35 @@ Iterate as needed until all configuration is correctly ported to your local envi
 
 ---
 
-## Step 6: Move to production
+## Step 6: Deciding on persistent vs ephemeral airflow database
+
+The Dagster Airflow migration tooling supports two methods for persisting Airflow metadatabase state. By default, it will use an ephemeral database that is scoped to every job run and thrown away as soon as a job run terminates. You can also configure a persistent database that will be shared by all job runs. This tutorial uses the ephemeral database, but the persistent database option is recommended if you need the following features:
+
+- `retry-from-failure` support in Dagster
+- Passing Airflow state between DAG runs (i.e., xcoms)
+- `SubDAGOperators`
+- Airflow Pools
+
+If you have complex airflow DAGs (cross-dag state sharing, pools) or the ability to retry-from-failure you will need to have a persistent database when making your migration.
```

Comment:
```suggestion
If you have complex Airflow DAGs (cross-DAG state sharing, pools) or the ability to retry-from-failure you will need to have a persistent database when making your migration.
```

❓ Should `retry-from-failure` be formatted as code? It is in the list preceding this paragraph - either way, we should be consistent